hudi-client/hudi-client-common/src/main/java/org/apache/hudi/client/transaction/lock/StorageBasedLockConfig.java

```diff
@@ -28,53 +28,54 @@
 import java.util.concurrent.TimeUnit;
 
 import static org.apache.hudi.common.config.HoodieCommonConfig.BASE_PATH;
```

Comment:
Maybe we should put the config in `org.apache.hudi.config` package.

Reply:
Which one, BASE_PATH?

Reply:
got it, i will do that